src/interface/src/app/signup/signup.component.html

```diff
@@ -1,104 +1,126 @@
 <div class="signup-root">
   <div class="info-card-container">
-    <div class="info-card">
-      <h3><b>With a Planscape account, you can:</b></h3>
-
-      <ul>
-        <li>
-          <h3>Save planning areas</h3>
-        </li>
-        <li>
-          <h3>Create plans, configurations, and scenarios</h3>
-        </li>
-        <li>
-          <h3>Download shapefiles & metadata</h3>
-        </li>
-        <li>
-          <h3>Share work with collaborators</h3>
-        </li>
-      </ul>
-
-      <h3><b>As a guest user, you can:</b></h3>
-
-      <ul>
-        <li>
-          <h3>Explore regional data on the map</h3>
-        </li>
-        <li>
-          <h3>View plans and/or scenarios that are publicly visible*</h3>
-        </li>
-      </ul>
-
-      <p><i>*Accessible only via direct links</i></p>
-    </div>
+    <app-info-card></app-info-card>
   </div>
 
   <div class="signup-form-container">
-    <div class="signup-form">
-      <h1 class="signup-title">Create your account</h1>
+    <div class="signup-form-subcontainer">
+      <div class="planscape-color-logo">
+        <img src="assets/svg/planscape-color-logo.svg" alt="Planscape logo" />
+      </div>
+      <div class="signup-form">
+        <h1 class="signup-title">Create your account</h1>
 
-      <form [formGroup]="form" (ngSubmit)="signup()">
-        <mat-form-field appearance="fill">
-          <mat-label>First name</mat-label>
-          <input type="text" required formControlName="firstName" matInput />
-        </mat-form-field>
+        <form [formGroup]="form" (ngSubmit)="signup()">
+          <mat-label class="standard-label"
+            >First Name
+            <span class="required-blurb">required</span>
+          </mat-label>
+          <mat-form-field appearance="outline">
+            <input type="text" required formControlName="firstName" matInput />
+          </mat-form-field>
+          <app-field-alert
+            *ngIf="
+              this.form.controls['firstName'].errors !== null &&
+              this.form.controls['firstName'].dirty === true
+            "
+            message="This field must contain a value."
+            title="First Name Error"></app-field-alert>
 
-        <mat-form-field appearance="fill">
-          <mat-label>Last name</mat-label>
-          <input type="text" required formControlName="lastName" matInput />
-        </mat-form-field>
+          <mat-label class="standard-label"
+            >Last Name
+            <span class="required-blurb">required</span>
+          </mat-label>
+          <mat-form-field appearance="outline">
+            <input type="text" required formControlName="lastName" matInput />
+          </mat-form-field>
+          <app-field-alert
+            *ngIf="
+              this.form.controls['lastName'].errors !== null &&
```

Comment:
`this` on the templates looks weird, not sure if this should be just `form...` or am I missing something?